### PR TITLE
Add title to 503 error page.

### PR DESCRIPTION
**Pull request by codybutz**
This is just a small (and maybe dumb) change, I know, but just an idea.

### CODE REVIEW

File: resources/views/errors/503.blade.php [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <html>
 	<head>
+        <title>Be right back.</title>
```
Comment (taiyaei) [inline resources/views/errors/503.blade.php]: Tabs

**Comment by taiyaei** [comment]
:+1:

File: resources/views/errors/503.blade.php (hunk 1 above)
Comment (mickaelandrieu) [inline resources/views/errors/503.blade.php]: "Service unavailable" sounds better (see RFC)

**Comment by taylorotwell** [comment]
Can this be sent to develop branch?

**Pull request closed without merging**